alsa-lib/tests/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -euo pipefail
+
+TESTDIR="$(dirname "${0}")"
+
+if [ -z "${1:-}" ]; then
+  echo "Usage: $0 FULLY_QUALIFIED_PACKAGE_IDENT"
+  exit 1
+fi
+
+TEST_PKG_IDENT="$1"
+
+hab pkg install --binlink core/bats
```

Comment:
`--binlink` should be at the end of this command

Reply:
Done - I'll change this throughout my currently open PRs now.